cores/arduino/delay.c

```diff
@@ -68,12 +68,15 @@ void delay( unsigned long ms )
     return ;
   }
 
-  uint32_t start = _ulTickCount ;
-
-  do
-  {
-    yield() ;
-  } while ( _ulTickCount - start < ms ) ;
+  uint32_t start = micros();
+
+   while (ms > 0) {
+     yield();
+     while ( ms > 0 && (micros() - start) >= 1000) {
```

Comment:
- indent is off by 1 space for all the function except the first line
- there is an extra space after the open parenthesis before `ms`

Reply:
@cmaglie removed the white space!